kotlin-jvm/src/test/kotlin/framework/atrium/Assertions.kt

```diff
@@ -0,0 +1,42 @@
+package framework.atrium
```

Comment:
The method names are ok-ish.

But please 
1. move the content of this class inside the class `testing.asserts.AtriumAsserts`
2. create a copy of this class in `testing.failing.FailingAtriumAsserts` where you make the tests fail so that we can see how clear the error messages are.